packages/gatsby-source-contentful/src/plugin-options.js

```diff
@@ -1,5 +1,6 @@
-const Joi = require(`@hapi/joi`)
+const joi = require(`joi`)
```

Comment:
Assuming we're using this file because it already exists, but perhaps that's a good example of what we could do. I'd still like to explore doing this outside of gatsby-node and developing a new convention.